tests/unit/igraph_motifs_randesu_estimate.out

```diff
@@ -8,7 +8,7 @@ Full graph of 50 vertices, motif size 3, sample all, (50 choose 3 = 19600):
 Estimate: 19600
 
 Full graph of 50 vertices, motif size 3, sample all, cut_prob 0.1 at each level:
-Estimate: 15990
+Estimate: 16034
```

Comment:
Any idea why the estimate has changed? Your changes should only affect the case of `cut_prob=NULL`, right? Is it because _earlier_ tests in the same test case took a different code path?

Reply:
Yes, I investigated this. Take a look at line 339 in the new version. This line isn't run anymore when using `NULL`, so there are fewer invocations to the RNG. This affects the subsequent random numbers that are generated.